packages/clay-badge/demo/App.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * © 2018 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
```

Comment:
I don't think we need to do the `* as` thing here (or in the other files either). If you search, you'll find a few discussions about what React exports (and what it "should export"); this is the current reality:

- https://github.com/facebook/react/blob/1d6b1660a29c452e954d053a14c5adc42ccf94ba/packages/react-dom/index.js#L14-L16
- https://github.com/facebook/react/blob/1d6b1660a29c452e954d053a14c5adc42ccf94ba/packages/react-dom/index.js#L14-L16

And the official docs ([ReactDOM](https://reactjs.org/docs/react-dom.html), [React](https://reactjs.org/docs/react-api.html)) just tell you to `import React`, `import ReactDOM`.

Reply:
The reason I use `* as React` was due to the ts compiler complaining.

![screen shot 2019-02-25 at 12 17 19 pm](https://user-images.githubusercontent.com/6843530/53365980-5b383a80-38f7-11e9-94e9-01d75c08ae30.png)

I looked into it and we would have to add this option to our tsconfig, `"allowSyntheticDefaultImports": true`. Not sure if that is the route we want to go though. 

I was reading these issues about it, https://github.com/Microsoft/TypeScript/issues/3337 and https://github.com/Microsoft/TypeScript-React-Starter/issues/8

Any preference on it?

Reply:
Ah, that's a bummer. I wanted to stick to the pattern in the React docs but `allowSyntheticDefaultImports` seems like a bit of a hack. So `* as React` is probably the least horrible option.

Reply:
Yeah thats what I figured as well. Ill just leave it as is for now, and then if for some reason TS fixes this, we can make the change everywhere.